posthog-node/index.ts

```diff
@@ -1 +1,2 @@
 export * from './src/posthog-node'
+export * from './src/extensions/sentry-integration'
```

Comment:
Unsure how you wanted this exported (or even how the package structure should work), so I tried to mirror `posthog-js`